### ISSUE TITLE
[*] Smart Fields - `option` parameter for a custom field is different when called by a listing view or a detail view 

### ISSUE THREAD

**Issue by AlexisNo** [ForestAdmin/forest-express-sequelize#139]
Hi,

I defined my relations using Sequelize `4.11.3`.
I have a custom field on a model `subscription` that has to show data from a deep relation.

    subscription.user.cards[0].number

This code is working on the listing view :

```javascript
Liana.collection('subscription', {
  fields: [{
    field: 'card',
    type: 'String',
    get: (subscription) => {
      return subscription.getUser({
        include: [{
          model: models.user_card,
          as: 'cards'
        }]
      })
      .then(user => {
        return user.cards[0].number
      })
    }
  }]
});
```

However, when I want to see the details of a subscription, I've got the following error:

    [forest] 🌳🌳🌳  Unexpected error: subscription.getUser is not a function

It seems that a `subscription.user` property is set with the raw content of the user, but the `Subscription` instance does not comes with the getter that would allow me to use the Sequelize relations to get my content.

I think the parameter of `get(object)` should be the same in both cases.

Is it correct?

**Comment by arnaudbesnier** [comment]
Hi @AlexisNo, the `subscription` parameter is the Sequelize instance for the current `subscription` but a hash containing its values.

So what you can do is something like:
```javascript
Liana.collection('subscription', {
  fields: [{
    field: 'card',
    type: 'String',
    get: (subscription) => {
      return models.user.findOne({
        where: { id: subscription.userId }, // TO ADAPT with the id you want to use.
        include: [{
          model: models.user_card,
          as: 'cards'
        }]
      })
      .then(user => {
        return user.cards[0].number
      })
    }
  }]
});
```

Does it work for you?

**Comment by AlexisNo** [comment]
Thanks @arnaudbesnier. That's what I'm currently using. :)

But on the listing view, it is possible to use `subscription.getUser()`. That is not a blocking issue thanks to this workaround, but I discovered that it was not working on the detailed view only later. I think it would be nice to have the listing view behavior for both.

**Comment by arnaudbesnier** [comment]
@AlexisNo the fix is available in the liana version v2.12.8.

🌲🌲🌲
